### PR TITLE
Feature/another results refactor

### PR DESCRIPTION
**Pull request by rnugent3**
I've refactored results based on team discussions. I need to add logic to fix some of the ways performance is computed, but first I'd like feedback on the structure/design of results things.

### CODE REVIEW

File: fda-model/ead/simulation.cs [hunk 1]
```diff
@@ -438,6 +429,11 @@ public SimulationBuilder withPerformanceMetrics(Results mr)
                 _sim._results = mr;
                 return new SimulationBuilder(_sim);
             }
+            public SimulationBuilder withAdditionalThreshold(Threshold threshold)
+            {
+                _sim._results.Thresholds.AddThreshold(threshold);
```
Comment (HenryGeorgist) [inline fda-model/ead/simulation.cs]: im not entirely sure, but you might need to check that thresholds is not null.

File: fda-model-test/AlternativeTest.cs [hunk 2]
```diff
@@ -4,7 +4,7 @@
 using System.Text;
 using System.Threading.Tasks;
 using Xunit;
-using ead;
+using compute;
```
Comment (HenryGeorgist) [inline fda-model-test/AlternativeTest.cs:7]: that is a good refactor given a compute does more than ead.

**Review by HenryGeorgist — APPROVED** [review]
looks good to me - will break @Brennan1994 in fda-viewmodel but only a little bit.

File: fda-model/ead/simulation.cs (hunk 1 above)
Comment (rnugent3) [reply fda-model/ead/simulation.cs]: Check if threshold is null or thresholds? 

We have tests passing with this structure, the simulation is being built with the additional threshold without hitting exceptions.

**Pull request merged**